src/main/java/ch/njol/skript/expressions/ExprPermissions.java

```diff
@@ -43,37 +45,45 @@
 @Description("Returns all permissions of the defined player(s). Note that the modifications to resulting list do not actually change permissions.")
 @Examples("set {_permissions::*} to all permissions of the player")
 @Since("2.2-dev33")
-public class ExprPermissions extends PropertyExpression<Player, String> {
+public class ExprPermissions extends SimpleExpression<String> {
 	
 	static {
-		register(ExprPermissions.class, String.class, "permissions", "players");
```

Comment:
The `s` after the apostrophe should be optional.

Reply:
permission?

all of the permission of player?

Why?

Reply:
Ohh, for the possessive plural form, I see now. Makes sense in English but kinda weirdly looking in Skript to me, is this really a standard?

Reply:
Yeah, it has always been like that.

Reply:
Good then I suppose 👀